src/database/schema.ts

```diff
@@ -61,3 +75,74 @@ export const sessionRelations = relations(session, ({ one }) => ({
     references: [users.uuid],
   }),
 }))
+
+export const jobs = pgTable('jobs', {
+  ...createdAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  user_uuid: uuid('user_uuid')
+    .notNull()
+    .references(() => users.uuid),
+  name: varchar('name'),
+  status: varchar('status').notNull(),
+  data: jsonb('data'),
+  errors: jsonb('errors'),
+})
+
+export const jobsRelations = relations(jobs, ({ one }) => ({
+  user: one(users, {
+    fields: [jobs.user_uuid],
+    references: [users.uuid],
+  }),
+}))
+
+export const tickets = pgTable('tickets', {
+  ...createdAt,
+  ...updatedAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  title: varchar('title'),
+  description: varchar('description'),
+  user_uuid: uuid('user_uuid').references(() => users.uuid),
+  creator_uuid: uuid('creator_uuid')
+    .notNull()
+    .references(() => users.uuid),
+})
+
+export const ticketsRelations = relations(tickets, ({ one, many }) => ({
+  ticketAssignee: one(users, {
+    fields: [tickets.user_uuid],
+    references: [users.uuid],
+    relationName: 'ticket_assignee',
+  }),
+  ticketCreator: one(users, {
+    fields: [tickets.creator_uuid],
+    references: [users.uuid],
+    relationName: 'ticket_creator',
+  }),
+  attachments: many(attachments),
+}))
+
+export const attachments = pgTable('attachments', {
+  ...createdAt,
+  ...updatedAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  ticket_uuid: uuid('ticket_uuid')
+    .notNull()
+    .references(() => tickets.uuid),
+  url: varchar('url'),
```

Comment:
Here I'd store the s3 key and bucket instead of the whole url - since the urls are 3rd party and can change if we migrate to Cloudflare r2 for cost optimization for example

Reply:
I'm using attachment `uuid` record as s3Key, so I think that I can remove url from the table and just return it in the response.